mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -546,6 +546,7 @@ vsphere_connections_dialog.guest_password = Guest Password
 bonjour.bonjour_services = Bonjour services
 bonjour.no_service_discovered = No service discovered
 bonjour.bonjour_disabled = Bonjour disabled
+bonjour.bonjour_initializing = Bonjour initializing...
```

Comment:
we use to add the `...` postfix to operations that trigger opening a dialog/window and it's not the case here. is it at least disabled?

Reply:
yes it is, it works the same as for `bonjour.bonjour_disabled`. Should I remove "..." ?

Reply:
yes please